components/grid/toolbar.md

```diff
@@ -10,282 +10,175 @@ position: 45
 
 # Grid Toolbar
 
-The grid provides a toolbar where you can add a variety of actions that are not tied to a concrete row.
+The [Blazor Grid](https://demos.telerik.com/blazor-ui/grid/overview) toolbar can render built-in and custom tools. This article describes the built-in tools and shows how to add custom tools or customize the toolbar.
 
-To use a toolbar, define the `GridToolBarTemplate` tag of the grid. In it, you can use arbitrary HTML and components to get the desired layout, and also `GridCommandButton` instances in (you can read more about the features available in those buttons in the [Command Column](slug://components/grid/columns/command) article).
+## Built-in Tools
 
->note The toolbar is not associated with an item from the data source. The `Item` field on the click event handler argument of a `GridCommandButton` will always be `null` and the `Edit`, `Update`, `Cancel` commands do not work with it.
+By default, the [Blazor Grid](https://demos.telerik.com/blazor-ui/grid/overview) displays all its built-in tools in the order below. Use the tool tag if you need to define a tool explicitly in a [custom toolbar configuration](#toolbar-configuration).
 
-In this article, you will learn how to use:
+### Command Tools
 
-* [Built-in Commands](#built-in-commands)
-* [Custom Commands](#custom-commands)
-* [Custom Layout](#custom-layout)
+@[template](/_contentTemplates/common/parameters-table-styles.md#table-layout)
 
+| Tool Name | Tool Tag | Description |
+| --- | --- | --- |
+| Add | `GridToolBarAddTool` | An add command that fires the [`OnAdd` event](slug://components/grid/editing/overview#events). |
+| CsvExport | `GridToolBarCsvExportTool` | An export command for CSV files that fires the [`OnBeforeExport` event](slug://grid-export-events#onbeforeexport). |
+| ExcelExport | `GridToolBarExcelExportTool` | An export command for Excel files that fires the [`OnBeforeExport` event](slug://grid-export-events#onbeforeexport). |
+| SearchBox | `GridToolBarSearchBoxTool` | A searchbox that filters multiple Grid columns simultaneously. |
 
-## Built-in Commands
+### Layout Tools
```

Comment:
Consider merging the two tool tables into one for simplicity.

Reply:
That would differ from the PDFViewer and ListBox articles.